schema/deploy/computed_columns/form_change_calculated_total_payment_amount_to_date.sql

```diff
@@ -0,0 +1,17 @@
+-- Deploy cif:computed_columns/form_change_calculated_total_payment_amount_to_date from pg
+
+begin;
+
+create or replace function cif.form_change_calculated_total_payment_amount_to_date(cif.form_change)
+returns numeric
+as
+$fn$
+
+-- TODO: placeholder
+select 0;
```

Comment:
This calculation depends on the Project Summary Form (#1399 ) and will be handled in https://github.com/bcgov/cas-cif/issues/1419